src/Microsoft.ML.LightGBM/LightGbmArguments.cs

```diff
@@ -321,6 +321,11 @@ public enum EvalMetricType
         [TGUI(Label = "Ranking Label Gain")]
         public string CustomGains = "0,3,7,15,31,63,127,255,511,1023,2047,4095";
 
+        [Argument(ArgumentType.AtMostOnce, HelpText = "Parameter for the sigmoid function. Used only in binary and multiclassova " +
```

Comment:
>multiclassova [](start = 115, length = 13)

Two points:

First, consider how FastTree is arranged. If an argument is only relevant to some trainer, then the way we accomplish that is by having that argument appear only in those specific argument classes that it is actually relevant. Is that possible to do here? This arrangement of "don't use this argument unless you're using this trainer" is really awkward, and not well arranged.

Even failing that, this word "multiclassova" is not one that I see is exposed to the user anywhere else, and we ought to therefore use a better description. `LightGbmMulticlassTrainer` perhaps. If you do that, please do `nameof` or something.